CRM/Speakcivi/Page/Speakcivi.php

```diff
@@ -520,7 +536,10 @@ public function determineActivityType($actionType) {
   /**
    * Determine which status to give to the activity
    */
-  public function determineActivityStatus($contact, $targetGroupId) {
+  public function determineActivityStatus($contact, $targetGroupId, $param): string {
```

Comment:
There was a mistake / unclarity in the specifications: surveys don't need special treatment for how the status is determined. You can keep the activity status that's determined based on consent status.

Reply:
ok. I've just reverted the commit.